tests/dialects/test_bigquery.py

```diff
@@ -845,6 +845,17 @@ def test_bigquery(self):
                 "duckdb": "SELECT CAST('09:05:03' AS TIME) - INTERVAL '2' HOUR",
             },
         )
+
+        self.validate_all(
+            "LOWER(CAST('HELLO' AS BYTES))",
+            write={
+                "bigquery": "LOWER(CAST('HELLO' AS BYTES))",
+                "duckdb": "CAST(LOWER(CAST(CAST('HELLO' AS BLOB) AS TEXT)) AS BLOB)",
+            },
+        )
+        self.validate_all(
+            "LOWER('HELLO')", write={"bigquery": "LOWER('HELLO')", "duckdb": "LOWER('HELLO')"}
```

Comment:
Styling nit: we prefer multi-line `read`/`write` dicts:

```suggestion
            "LOWER('HELLO')",
            write={
                "bigquery": "LOWER('HELLO')",
                "duckdb": "LOWER('HELLO')",
            },
```